docs/howto/UPGRADING.md

```diff
@@ -0,0 +1,93 @@
+# How to upgrade from previous versions
+
+### Creating a Custom Base Adapter
+
+You can upgrade to ams ``.10`` by creating a custom Base Adapter to
+emulate the response that your application currently serves.
+
+###### Base Adapter
+
+A base adapter takes a ``ActiveModel::Serializer`` or
+``ActiveModel::Serializer::ArraySerializer`` instance, and creates a hash
+used for serialization in its ``serializable_hash`` method. A base adapter
+could look like this:
+
+```ruby
+class MyApp::BaseAdapter < ActiveModel::Serializer::Adapter::Base
```

Comment:
Code blind 
Semicolon in place of colon

Way i wrote was ambiguous 

B mobile phone

> On Oct 22, 2015, at 12:02 PM, Nick <notifications@github.com> wrote:
> 
> In docs/howto/UPGRADING.md:
> 
> > +# How to upgrade from previous versions
> > +
> > +### Creating a Custom Base Adapter
> > +
> > +You can upgrade to ams ``.10`` by creating a custom Base Adapter to
> > +emulate the response that your application currently serves.
> > +
> > +###### Base Adapter
> > +
> > +A base adapter takes a ``ActiveModel::Serializer`` or
> > +``ActiveModel::Serializer::ArraySerializer`` instance, and creates a hash
> > +used for serialization in its ``serializable_hash`` method. A base adapter
> > +could look like this:
> > +
> > +```ruby
> > +class MyApp::BaseAdapter < ActiveModel::Serializer::Adapter::Base
> @bf4:
> 
> typo: ;:. why not try adding this code with tests?
> 
> sorry I must be blind, I don't see the typo.
> 
> —
> Reply to this email directly or view it on GitHub.
>